_episodes/44.md

```diff
@@ -0,0 +1,213 @@
+     time:
+---
+
+## Hosts
+
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+
+## Guests
+
+* [Luis Paolini](https://www.linkedin.com/in/luispaolini/), Product Advocate at
+  [Metabase](https://www.metabase.com/)
+* [Andrew DiBiasio](https://www.linkedin.com/in/andrewdibiasio/), Software
+  Engineer at [Starburst](https://starburst.io)
+* [Piotr Leniartek](https://www.linkedin.com/in/piotrleniartek), Product Manager
+  at [Starburst](https://starburst.io)
+
+## Recap of Trino in 2022
+
+Highlights from the blog post [The rabbit reflects on Trino in 2022]({% post_url
+2023-01-10-trino-2022-the-rabbit-reflects %}):
+
+* 
+
+## Releases 404 to 406
+
+Official highlights from Martin Traverso:
+
+[Trino 404]({{site.url}}/docs/current/release/release-404.html) not found
+
+[Trino 405]({{site.url}}/docs/current/release/release-405.html)
+
+* Support for `ALTER COLUMN ... SET DATA TYPE` statement.
+* Support for Apache Arrow when reading from BigQuery.
+* Support for views in the Delta Lake connector.
+* Support for the Iceberg REST catalog.
+* Support for Protobuf encoding in the Kafka connector.
+* Support for fault-tolerant execution in the MongoDB connector.
+* Support for `DELETE` and query pushdown in the Redshift connector.
+* Performance improvements when reading Parquet data.
+
+[Trino 406]({{site.url}}/docs/current/release/release-406.html)
+
+* tbd
+
+Further thoughts and highlights from Cole:
+
+* tbd
+
+More detailed information is available in the release notes for
+[Trino 405]({{site.url}}/docs/current/release/release-405.html),
+and
+[Trino 406]({{site.url}}/docs/current/release/release-406.html).
+
+We also shipped trino-python-client 0.321.0 with the following improvements:
+
+* Add support for SQLAlchemy 2.0.
+* Add support for `varbinary` query parameters.
+* Add support for variable precision `datetime` types.
+
+## What is Metabase
+
+[Metabase](https://www.metabase.com/) is the BI tool with the friendly UX and
+integrated tooling to let your company explore data on their own.
+
+Core features and advantages of Metabase include the following:
+
+* Visual query build
+* Dashboards
+* Models
+
+Metabase is a web-based application that you run on a server and make it
+available for multiple users. It uses SQL to create queries, reports,
+visualizations, dashboards and more.
+
+Metabase is an open source project licensed under the GNU Affero General Public
+License (AGPL) license. It is written in Clojure and therefore runs on the Java
+virtual machine.
+
+Metabase is also the name of the company, founded in 20xx. It provides an
+expanded version under a commercial license, a SaaS version of the application,
+support and others services, and manages the open source project.
+
+## History of Metabase and Trino
```

Comment:
I will start this topic

But ask questions to @paoliniluis  and @andrewdibiasio6 and @leniartek as the conversation goes.